pandas/tests/window/test_window.py

```diff
@@ -1,6 +1,6 @@
 from collections import OrderedDict
 from datetime import datetime, timedelta
-from itertools import product
+from typing import Any
```

Comment:
Not opposed to this but note that we haven't done annotations in any other test module nor do we check it (mypy.ini excludes these) so would be OK to leave out for now as well